tests/countries/test_bhutan.py

```diff
@@ -0,0 +1,114 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS.md file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from unittest import TestCase
+
+from holidays.countries.bhutan import Bhutan, BT, BTN
+from tests.common import CommonCountryTests
+
+
+class TestBhutan(CommonCountryTests, TestCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass(Bhutan, years=(range(1970, 2050)))
+
+    def test_country_aliases(self):
+        self.assertAliases(Bhutan, BT, BTN)
+
+    def test_no_holidays(self):
+        self.assertNoHolidays(Bhutan(years=1969))
+
+    def test_nyilo(self):
+        self.assertHolidayName("ཉི་ལོག་", (f"{year}-01-02" for year in range(1970, 2050)))
+
+    def test_kings_birthday_day1(self):
+        self.assertHolidayName(
+            "རྒྱལ་པོའི་འཁྲུངས་སྐར།༼ཉིན༡༽།", (f"{year}-02-21" for year in range(1970, 2050))
+        )
+
+    def test_kings_birthday_day2(self):
+        self.assertHolidayName(
+            "རྒྱལ་པོའི་འཁྲུངས་སྐར།༼ཉིནམ་གཉིས་པ།༽", (f"{year}-02-22" for year in range(1970, 2050))
+        )
+
+    def test_kings_birthday_day3(self):
+        self.assertHolidayName(
+            "རྒྱལ་པོའི་འཁྲུངས་སྐར།༼ཉིན་གསུམ་པ།༽", (f"{year}-02-23" for year in range(1970, 2050))
+        )
+
+    def test_birth_anniversary_third_druk_gyalpo(self):
+        self.assertHolidayName(
+            "གསུམ་པ་དབུ་མའི་འཁྲུངས་སྐར་དུས་ཆེན།", (f"{year}-05-02" for year in range(1970, 2050))
+        )
+
+    def test_coronation_king_jigme_khesar(self):
+        self.assertHolidayName(
+            "རྒྱལ་པོ་འཇིགས་མེད་ཁེ་སར་རྣམ་རྒྱལ།ཀོ་རོ་ན་ཤི་ཡ།", (f"{year}-11-01" for year in range(1970, 2050))
+        )
+
+    def test_birth_anniversary_fourth_druk_gyalpo(self):
+        self.assertHolidayName(
+            "དྲུག་རྒྱལ་བཞི་པའི་འཁྲུངས་སྐར་དུས་ཆེན།", (f"{year}-11-11" for year in range(1970, 2050))
+        )
+
+    def test_national_day(self):
+        self.assertHolidayName("རྒྱལ་ཡོངས་དུས་ཆེན།", (f"{year}-12-17" for year in range(1970, 2050)))
+
+    def test_losar(self):
+        self.assertHolidayName("ལལོ་གསར་", ("2022-03-03", "2024-02-11", "2025-02-28"))
+
+    def test_blessed_rainy_day(self):
+        self.assertHolidayName("ཁྲུས་འབབས་ཀྱི་ཉིནམ།", ("2022-09-23", "2024-09-23", "2025-09-23"))
+
+    def test_dashain(self):
+        self.assertHolidayName("དྲང་བའི་བདུད་རྩིའི་ཉིམ།", ("2022-10-05", "2024-10-12", "2025-10-02"))
```

Comment:
Need to check latest 5-6 years. And check for name presence in whole years range.